src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileValidation.java

```diff
@@ -302,6 +322,258 @@ public ValidationOutputData validateMandatoryFieldsAndData(PerformanceProfile pe
 
         return validationOutputData;
     }
+
+    /**
+     * Validates the data present in the metric profile object before adding it to the map
+     * @param metricProfile
+     * @return
+     */
+    private ValidationOutputData validateMetricProfileData(PerformanceProfile metricProfile) {
+        // validate the mandatory values first
+        ValidationOutputData validationOutputData = validateMandatoryMetricProfileFieldsAndData(metricProfile);
+
+        // If the mandatory values are present,proceed for further validation else return the validation object directly
```

Comment:
enclose from here to end with try catch
and catch block add this
    validationOutputData.setSuccess(false);
                    validationOutputData.setMessage(errorString.toString());

Reply:
done